code/obj/machinery/floorflusher.dm

```diff
@@ -18,6 +18,7 @@
 	var/flush = 0	// true if triggered
 	var/obj/disposalpipe/trunk/trunk = null // the attached pipe trunk, if none reject user
 	var/flushing = 0	// true if flushing in progress
+	var/max_capacity = 100
```

Comment:
why not use the define you made

Reply:
LOG_FLUSHING_THRESHOLD is higher by a magnitude. This instead mirrors the maximum capacity of a floor flusher, which mirrors the maximum capacity of a crate/locker.

Reply:
and why is that not a define

Reply:
Because this is a quick, dirty fix to prevent lag machines and I wouldn't know where to even put it for now. I am actively working to refactor disposals, and the defines will be placed in the component for flushers when that is made.